android/build.gradle

```diff
@@ -6,7 +6,7 @@ buildscript {
         google()
     }
     dependencies {
-        classpath 'com.android.tools.build:gradle:3.1.0-alpha09'
+        classpath 'com.android.tools.build:gradle:3.1.3'
```

Comment:
This seems out of scope to me? Oh well.

Reply:
Android Studio really likes to change it. I can pull it out.